tools/wavetool.py

```diff
@@ -205,6 +212,29 @@ def analyze_wav_wov(wave, fs):
         sys.exit(1002)
     print('wave analysis result: PASSED')
 
+def analyze_wav_alsabat(wave, fs_wav):
+    # Drop the ramp up data in unstable state
+    time_skip = 0.25
```

Comment:
What is this value used here? Global value?

Reply:
There is a ramp in recorded wave in alsabat test case, and the duration is about 0.25 second